doc/source/tutorial/stats/continuous_laplace_asymmetric.rst

```diff
@@ -0,0 +1,20 @@
+
+.. _continuous-laplace:
+
+Asymmetric Laplace Distribution
+================================================================
+
+.. math::
+   :nowrap:
+
+    \begin{eqnarray*} f(x, \kappa) &=& \frac{1}{\kappa+\kappa^{-1}}\exp(-x/\kappa),\quad x\ge0;\\
+                     &=& \frac{1}{\kappa+\kappa^{-1}}\exp(x\kappa),\quad x<0.\end{eqnarray*}
+
+.. math::
+   :nowrap:
+
+    \begin{eqnarray*} \mu &=& \kappa^{-1}-\kappa\\
+    \sigma^2 &=& \kappa^{-2}-\kappa^2\\
+    \tilde\mu_3 &=& \frac{2(1-\kappa^6)}{(1+\kappa^4)^{3/2}}\end{eqnarray*}
+
+Implementation: `scipy.stats.laplace_asymmetric`
```

Comment:
Can you add a References section here prior to the Implementation? (E.g. see doc/source/tutorial/stats/continuous_kstwobign.rst for an example)
Ideally some which are readily accessible.  The reference in the implementation is behind a paywall.

Reply:
Thanks done.